vsintegration/update-vsintegration.cmd

```diff
@@ -138,15 +138,6 @@ if /i "%PROCESSOR_ARCHITECTURE%"=="AMD64" (
     %SN64% -Vr VisualFSharp.Salsa,b03f5f7f11d50a3a
 )
 
-if exist "%ProgramFiles(x86)%\Microsoft Visual Studio %VisualStudioVersion%\Common7\IDE\PrivateAssemblies" (
```

Comment:
I was kind of hoping this whole file might disappear - that the way you update the VS integration is to install the VSIX?